l3kernel/l3luatex.dtx

```diff
@@ -219,6 +243,31 @@
   { \lua_shipout_e:n { \exp_not:n {#1} } }
 \cs_new:Npn \lua_escape:e #1 { \@@_escape:n {#1} }
 \cs_new:Npn \lua_escape:n #1 { \lua_escape:e { \exp_not:n {#1} } }
+%    \end{macrocode}
+% \end{macro}
+% \end{macro}
+% \end{macro}
+%
+% \begin{macro}{\lua_load_module:n}
+%   Wrapper around |ltx.utils.require'|\meta{module}|'|.
+%    \begin{macrocode}
+\cs_generate_variant:Nn \bool_if:nF { e }
```

Comment:
`\bool_if:nTF` already evaluates the argument, so I don't think that `e` type expansion is needed here.

Reply:
Correct. Fixed